node/package.json

```diff
@@ -12,23 +12,25 @@
     "ramda": "^0.25.0"
   },
   "devDependencies": {
-    "@types/atob": "^2.1.2",
-    "@types/bluebird": "^3.5.25",
-    "@types/co-body": "0.0.3",
-    "@types/jest": "^27.0.2",
-    "@types/jsonwebtoken": "^8.5.0",
+    "@types/co-body": "^0.0.3",
+    "@types/jest": "^24.0.18",
     "@types/node": "^12.0.0",
     "@types/ramda": "types/npm-ramda#dist",
     "@vtex/api": "6.45.15",
-    "@vtex/prettier-config": "^0.3.1",
-    "@vtex/tsconfig": "^0.6.0",
-    "tslint": "^5.12.0",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-config-vtex": "^2.1.0",
+    "@vtex/test-tools": "^1.0.0",
+    "@vtex/tsconfig": "^0.5.6",
+    "casual": "^1.6.2",
+    "ts-mockito": "^2.6.1",
     "typescript": "3.9.7",
     "vtex.storefront-permissions": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.storefront-permissions@1.33.2/public/@types/vtex.storefront-permissions"
   },
+  "resolutions": {
+    "@types/testing-library__dom": "6.12.1",
+    "@types/express": "4.16.0",
+    "@types/express-serve-static-core": "4.16.0"
+  },
```

Comment:
Can you explain why you needed these? What kind of errors were you getting? Just trying to see if we can do without.

Reply:
I had some troubles with the Typescript compilation on lint execution, mainly for the `@types/express-serve-static-core`, so I checked the template and there is this configuration [there](https://github.com/vtex-apps/service-example/blob/main/node/package.json#L16).

Reply:
Got it, probably due to the craziness of VTEX IO's outdated deps... tslint is also something not much used nowadays, people tend to use eslint directly. But let's not touch this now. It's fine to add this here to make it work now.